stacks/wordpress/uninstall.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# chart
+################################################################################
+STACK="wordpress"
+NAMESPACE="wordpress"
+
+
+helm uninstall "$STACK" \
+  --namespace "$NAMESPACE"
```

Comment:
lets add `kubectl delete ns wordpress` here as well :D

Reply:
Applied the suggestion, thank you